pcsx2/gui/FrameForGS.cpp

```diff
@@ -29,6 +29,9 @@
 #include <sstream>
 #include <iomanip>
 
+int	image_width;
+int	image_height;
```

Comment:
Probably should make this static and use the `s_` prefix.

I'd prefer a space instead of a tab too.